THIRD-PARTY-NOTICES

```diff
@@ -86,3 +86,48 @@ https://github.com/mongodb/mongo-csharp-driver.
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
      See the License for the specific language governing permissions and      
      limitations under the License.                                             
+
+3. The following files: NativeBufferInfo.cs, Zstandard64NativeMethods.cs, ZstandardNativeWrapper.cs, ZstandardStream.cs
+
+   Original work:
+     Copyright (c) 2016-present, Facebook, Inc. All rights reserved.
+
+     Redistribution and use in source and binary forms, with or without modification,
```

Comment:
I believe this means  that we'll need to include this notice in our MongoDB.Driver.Core package. I think the simplest way would be to include the entire THIRD-PARTY-NOTICES file with something like this.

```
  <ItemGroup>
    <None Include="..\..\License.txt" Pack="true" PackagePath="$(PackageLicenseFile)" />
    <None Include="..\..\License.txt" Pack="true" PackagePath="\" />
  </ItemGroup>
```

Reply:
@vincentkam can you please check whether I understood you correctly or no: https://github.com/DmitryLukyanov/mongo-csharp-driver/blob/8dd778298f90d573ca01d192e7f4da52339ea5c2/src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj#L129

Reply:
Oops yes, I messed up my snippet :) Your snippet looks correct!